recipe/patches/0002-no-python-blosc2-dep.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/pyproject.toml b/pyproject.toml
+index b503ab65..49d33800 100644
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -72,7 +72,9 @@ dependencies = [
+     "numexpr >= 2.6.2",
+     "packaging",
+     "py-cpuinfo",
+-    "blosc2 >= 2.3.0",
```

Comment:
we could use `c-blosc2` in place of `blosc2`

```suggestion
+    "c-blosc2 >= 2.3.0",
```

Reply:
c-blosc2 is a shared library, I don't think the package has the required metadata for pip check.

Reply:
ah cool, then ignore my comment, thanks for the explanation